README.md

```diff
@@ -1,70 +1,44 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] On your readme file, according to the [Microverse readme template](https://github.com/microverseinc/readme-template), kindly add the `acknowledgments` missing section.

That way, we will meet this particular requirement. 💪